### PR TITLE
Updated homepage, cart and footer

### PR DESCRIPTION
**Pull request by tanveertkd**
[Home](https://deploy-preview-4--ti-gear-shop.netlify.app/)  
[Cart](https://deploy-preview-4--ti-gear-shop.netlify.app/routes/cart.html)

Updated the homepage, footer and the mobile view of cart.

### CODE REVIEW

**Review by palash23jain — APPROVED** [review]
good use of CSS variables. quite neat and clean. good to go

File: css/cart.css [hunk 1]
```diff
@@ -208,15 +208,35 @@
...
         justify-content: center;
         align-items: center;
-        padding: 1rem;
+        padding: 0;
     }
 
     .cart-item-card {
-        flex-direction: column;
-        width: max-content;
+        /* flex-direction: column; */
```
Comment (msdeshmukh009) [inline css/cart.css]: Remove comments if not required.

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good.

File: css/main.css [hunk 2]
```diff
@@ -144,26 +177,38 @@ body {
     margin: 1rem auto;
 }
 
+.card-image {
+    width: 80%;
+}
+
 .card-body-content {
     display: flex;
```
Comment (someshtiwarii) [inline css/main.css:185]: Can use prettier for consistent spacing.

File: css/reset.css [hunk 3]
```diff
@@ -1,8 +1,3 @@
-/* * {
-    margin: 0;
```
Comment (someshtiwarii) [inline css/reset.css:2]: Comments can be removed if you want.

**Review by someshtiwarii — APPROVED** [review]
Clean code! Looks good to merge !

**Pull request merged**